Gemfile

```diff
@@ -1,4 +1,5 @@
 source 'https://rubygems.org'
+ruby "2.4.1"
```

Comment:
@Xiaohong-Deng That is a non-common use of the 'develop/development' branch. I'll change the PR to 'master' but might like to study some of the common git workflows for future projects.